Utils.py

```diff
@@ -6,6 +6,9 @@
 
 import os
 import FreeCAD
+import zipfile
+from PySide2.QtGui import QPixmap
+modPath = os.path.dirname(__file__).replace("\\", "/")
```

Comment:
Same comment: https://github.com/Ondsel-Development/Ondsel-Lens/pull/36#pullrequestreview-1481932139

Reply:
This is a widespread topic it appears. So maybe it can be changed in all files in a later PR to avoid conflicts.